scripts/Makefile

```diff
@@ -140,7 +140,7 @@ testsuite:
 # It assumes that some covrage reports already exists, otherwise it will fail
 .PHONY: coverage-report
 coverage-report:
-	python ../libbeat/scripts/aggregate_coverage.py -o ./${COVERAGE_DIR}/full.cov ./${COVERAGE_DIR}
+	python ${GOPATH}/src/github.com/elastic/beats/dev-tools/aggregate_coverage.py -o ./${COVERAGE_DIR}/full.cov ./${COVERAGE_DIR}
```

Comment:
Hey @ruflin, I just spot that the current version of `update.sh` would not update scripts/Makefile in a beat [1]. Any reason not to do so?

[1]: https://github.com/elastic/beats/blob/7e4b570ce34771ce7456443f8fbfe2de6731b365/libbeat/scripts/update.sh